engines/query-sparql/test/QuerySparql-test.ts

```diff
@@ -790,6 +791,70 @@ describe('System test: QuerySparql', () => {
     });
   });
 
+  describe('Comunica sparql regression test for #1029', () => {
```

Comment:
The issue reference might be a bit confusing, as it's not immediately clear as to what the issue is.
Can we move this test into the `query` describe block?

Reply:
i've moved it, but the description prolly needs some updating still. not resolving this thread until i finish that.